packages/amplify-category-predictions/package.json

```diff
@@ -8,11 +8,9 @@
     "amplify-category-auth": "1.7.1",
     "chalk": "^2.4.1",
     "chalk-pipe": "^2.0.0",
-    "commitizen": "^3.0.2",
```

Comment:
Why re we removing this? @SwaySway  Is this not getting used?

Reply:
Because the root ```package.json``` has it and it is configured there.

https://github.com/aws-amplify/amplify-cli/blob/master/package.json#L57